data/tutorials/recommendation/recommendation-test.md

```diff
@@ -0,0 +1,6 @@
+tutorials:
+  - title: "tutorial1"
+    slug: "to be added"
+    recommended_next_tutorials: 
+      -tutorial2 
+      -tutorial3
```

Comment:
Instead of creating a new file `recommendation-test.md`, I would suggest going to the "Tour of OCaml" tutorial and adding the "Your First OCaml Program" as recommendation.